src/test/java/newProje/tests/US_002_NotRegisterAgain.java

```diff
@@ -0,0 +1,95 @@
+package newProje.tests;
+
+import com.github.javafaker.Faker;
+import newProje.pages.HomePage;
+import newProje.utilities.ConfigReader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static newProje.utilities.Driver.getDriver;
+
+public class US_002_NotRegisterAgain {
+    HomePage homePage = new HomePage();
+    Faker faker = new Faker();
+    @Test
+    public void testUS02TC01() throws InterruptedException {
+        // Web sitesine git
+        getDriver().get(ConfigReader.getProperty("alloverCommerceUrl"));
+        // register linkine  tikla
+        homePage.register.click();
+        // username alanina kayitli bir veri gir
+        homePage.userName.sendKeys("gecerliUsername");
+        // email adress alanina bir veri gir
+        String Email = faker.internet().emailAddress();
+        homePage.email.sendKeys(Email);
+        //password alanina bir veri gir
+        String password = faker.internet().password();
+        homePage.password.sendKeys(password);
+        //I agree to the privacy policy kontrol kutusunu sec
+        homePage.policy.click();
+        //Sign Up butonuna tikla
+        homePage.submit.click();
+        //Anasayfa acilmadigini ve "An account is already registered with that username. Please choose another.
+        //" uyarisi gorulmus oldugunu dogrula
+        String alertMessage = homePage.alreadyRegisteredError.getText();
+        Assert.assertEquals(alertMessage, "An account is already registered with your email address. Please log in.");
```

Comment:
failed for me 
![image](https://github.com/ayseyttnc/allowercommerce/assets/67800186/bb77d00a-8082-4046-af77-cc7928880a5c)